### ISSUE TITLE
Cant register new account

### ISSUE THREAD

**Issue by buism** [phoronix-test-suite/openbenchmarking#22]
Tried multiple times, unknown error occurred. Not receiving an email

**Comment by Chris2000SP** [comment]
Same here. have a google address. I tried to disable my blocker addons, no luck.

**Comment by pbronez** [comment]
Same problem, not receiving account confirmation email at gmail address.

**Comment by Nobutarou** [comment]
Similar problem. I registered. No email come to my gmail address. I can login via web browser. However, I can never login via `phoronix-test-suite openbenchmarking-login`.  It says,

```
OpenBenchmarking.org Password:
    [PROBLEM] Account Not Found
```

**Comment by chomwitt** [comment]
same problem

**Comment by Morgonaut-Teresa** [comment]
I also cannot register on https://openbenchmarking.org/register. Did that today 20 times, from different computers, different IPs, different OSes, different browsers :( Nothing worked. Never received any email...

**Comment by perrin4869** [comment]
not receiving confirmation email either, checked spam too...

**Comment by Morgonaut-Teresa** [comment]
> 

yeah, that's for sure problem on their side. It's sad, because PTS has great potencial.

**Comment by SpokeyWheeler** [comment]
:-(
